packages/lib/services/person.ts

```diff
@@ -100,16 +101,20 @@ export const getPersonCached = async (personId: string) =>
     }
   )();
 
-export const getPeople = cache(async (environmentId: string): Promise<TPerson[]> => {
+export const getPeople = cache(async (environmentId: string, page: number = 1): Promise<TPerson[]> => {
   validateInputs([environmentId, ZId]);
   try {
+    const itemsPerPage = PERSONS_PER_PAGE;
     const personsPrisma = await prisma.person.findMany({
       where: {
         environmentId: environmentId,
       },
       select: selectPerson,
+      take: itemsPerPage,
+      skip: itemsPerPage * (page - 1),
     });
-    if (!personsPrisma) {
+
+    if (!personsPrisma || personsPrisma.length === 0) {
```

Comment:
please remove this block and just return an empty array when no people are found

Reply:
done!